src/ctapipe/tools/make_irf.py

```diff
@@ -279,20 +279,19 @@ def setup(self):
             "Aeff fov offset",
             raise_error=self.range_check_error,
         )
-        if self.full_enclosure:
```

Comment:
I don't think it makes sense to calculate the PSF for point-like irfs, since point-like irfs assume you know the position of your source anyway and the theta cut also truncates the PSF, no?

Reply:
The PSF simply measures how far away from the true position photons are being reconstructed, so it can always be defined. 

What matters for the PSF is the difference between diffuse simulations and point-like, since for the latter we only known of the behaviour for photons coming from a single position, so we can't talk about what happens when the source is placed in other parts of the field of view. Note that this is an entirely separate issue from the full_enclosure request. 

It is true that for point-like irfs there is a cut placed on how far away from the source position photons can be while being included, but you still need to know the spatial distribution of the the photons that are kept, so you need the PSF.

Reply:
No, the PDF is not used in point like analyses.

Reply:
See e.g.: https://github.com/gammapy/gammapy/blob/011cefef8916ebc9584fa06f52d999ca01af78ff/gammapy/data/data_store.py#L23

Reply:
It is even stronger than "we don't use it". The whole reason we do point-like IRFs is to simplify the IRF to have one less component: the PSF:

See section A.2 of https://redmine.cta-observatory.org/dmsf/files/17785/view

Reply:
It's not used for analysis in point-like IRFs, but it is used for visualization. E.g.if you produce a point-like excess map, you need to overplot the PSF to show how the map is correlated. Therefore it is always useful to include the PSF in even the point-like IRFs.  It's also useful just for cross-checking that your RAD_MAX looks ok.